tests/test_assertions.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+    proxy.py
+    ~~~~~~~~
+    ⚡⚡⚡ Fast, Lightweight, Pluggable, TLS interception capable proxy server focused on
+    Network monitoring, controls & Application development, testing, debugging.
+
+    :copyright: (c) 2013-present by Abhinav Singh and contributors.
+    :license: BSD, see LICENSE for more details.
+"""
+from typing import Any
+
+
+class Assertions:
+
+    def assertTrue(self, obj: Any) -> None:
```

Comment:
Why would you do something like this? It's quite unnatural for pytest.

Reply:
Will deprecate this eventually, I didn't want to add it, but I also wanted to keep my focus on async changes rather than removal of all `self.assert*` methods.  Will revisit all this mess again

Reply:
Tracking here https://github.com/abhinavsingh/proxy.py/issues/775

Thought I'll get this in, but let's keep for now.  Might address soon anyways.